### PR TITLE
Add properties file for configuring stockbot behavior

### PR DESCRIPTION
**Pull request by stamhankar999**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
+stockbot.properties
 env
+venv
```
Comment (curiousrohan1) [inline .gitignore:3]: What is this?

File: stockbot.properties.template [hunk 2]
```diff
@@ -0,0 +1,2 @@
+[main]
+dbFile=<Absolute path to stockbot sqlite3 db>
```
Comment (curiousrohan1) [inline stockbot.properties.template:2]: What is an absolute  path?

**Review by curiousrohan1 — COMMENTED** [review]
Looks good overall, but I have some questions.

File: .gitignore (hunk 1 above)
Comment (stamhankar999) [reply .gitignore:3]: When I create my virtual environment in Python, it is a convention to call it venv.

File: stockbot.properties.template (hunk 2 above)
Comment (stamhankar999) [reply stockbot.properties.template:2]: An absolute path starts from the root of the file system (e.g. /home/rohan/data/stocket). A relative path is evaluated relative to the current directory.

**Pull request merged**